<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keysight_e5270b_config_channel</class>
 <widget class="QWidget" name="keysight_e5270b_config_channel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_currComp"/>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_voltComp"/>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Voltage Compliance</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBox_currMeasRange"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Voltage Range</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Current Range</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Voltage Source</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Current Measurement Range</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Source</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Current Compliance</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Voltage Measurement Range</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBox_voltRange"/>
   </item>
   <item row="6" column="3">
    <widget class="QComboBox" name="comboBox_voltMeasRange"/>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="comboBox_currRange"/>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="radioButton_highSpeedADC">
     <property name="text">
      <string>High Speed ADC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_outputFilter">
     <property name="text">
      <string>Output Filter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="radioButton_highResADC">
     <property name="text">
      <string>High Res ADC</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_channel_active">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Channel active</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
